taskotron_python_versions/requires.py

```diff
@@ -38,25 +38,18 @@ def get_packages_by(self, **kwargs):
         """
         try:
             return self.query.filter(**kwargs).run()
-        except AttributeError:
+        except AttributeError as ae:
+            log.debug(repr(ae))
```

Comment:
I seriously don't know why we let AttributeError go, but I don't want to break anything. However this can cover serious problems (it did).

Reply:
I believe this is for rare cases when `self.query` is None, because `get_dnf_query` [fails to add repo](https://github.com/fedora-python/taskotron-python-versions/blob/develop/taskotron_python_versions/requires.py#L79). But it should not let the error go, it should probably somehow nicely say it can't complete the check.

Reply:
Oh. I've tried to make this bit better in 16ca638

Reply:
Thank you!